docs/reference/security/README.md

```diff
@@ -2,62 +2,6 @@
 
 This page describes Knative security and disclosure information.
 
-## Knative threat model
-
-* [Threat model](https://github.com/knative/community/blob/main/working-groups/security/threat-model.md)
-
-## Code Signature Verification
```

Comment:
I moved this to `verifying-cli.md`, as it didn't really fit with the rest of the overview